admin-frontend/app_singleapp/lib/widgets/features/table-expanded-view/create-strategy-widget.dart

```diff
@@ -126,6 +97,16 @@ class _CreateValueStrategyWidgetState extends State<CreateValueStrategyWidget> {
                     }
                     return null;
                   })),
+              if(rolloutStrategyAttributeList.isNotEmpty) Column (children: [for(var rolloutStrategyAttribute in rolloutStrategyAttributeList )
+                AttributeStrategyWidget(attribute: rolloutStrategyAttribute)]),
+              AttributeStrategyWidget(attributeStrategyFieldName: 'custom',), //we need to show this only when "Add custom attribute" button is clicked. Maybe it should be a stream of strategies we are about to add? We need to be able to remove them too
```

Comment:
yes, the list should be in the bloc and updates to it should be a stream wrapped around this section.